services/simplified_crontab

```diff
@@ -90,3 +90,8 @@ HOME=/var/www/circulation
 0 6 * * * root core/bin/run odl_bibliographic_import_monitor >> /var/log/cron.log 2>&1
 0 6 * * * root core/bin/run odl_consolidated_copies_monitor >> /var/log/cron.log 2>&1
 0 */8 * * * root core/bin/run odl_hold_reaper >> /var/log/cron.log 2>&1
+
+# Odilo
+#
+*/15 * * * * root core/bin/run odilo_monitor_full >> /var/log/cron.log 2>&1
```

Comment:
FullOdiloCollectionMonitor.INTERVAL_SECONDS (which we don't really use anymore, but it's a good guide) says this should be run every 4 hours. Fifteen minutes for the 'recent' monitor is good, I think.

Reply:
Updated!